packages/admin-client/src/common/components/PdfButtonField/PdfButtonField.tsx

```
@@ -19,10 +19,46 @@ const PdfButtonField: React.FC<PDFButtonFieldProps> = ({ record, resource }) =>
                 id: record?.id,
                 formType: formType
             })
-            .then(({ result }) => {
-                const filename =
-                    (resource === "applications" ? "application_" : "claim_") + record?.form_confirmation_id + ".pdf"
-                downloadPdf(result, filename)
+            .then(async ({ result }) => {
+                if (resource === "claims") {
+                    const employerInfo = await dataProvider
+                        .getEmployerInfo("employer", { id: record?.id })
+                        .then(async ({ result }) => {
+                            return result
+                        })
+                    const claimPeriodStartString = employerInfo["periodStart"]
```

You should check to see if `employerInfo` is null first before working with it (if it's null you can throw an error)

---

Looks like this block of code needs a `try/catch` as well